deploy/resolvers/00_deploy_eth_owned_resolver.ts

```diff
@@ -0,0 +1,22 @@
+import { DeployFunction } from 'hardhat-deploy/types'
+import { HardhatRuntimeEnvironment } from 'hardhat/types'
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { getNamedAccounts, deployments } = hre
+  const { deploy } = deployments
+  const { deployer } = await getNamedAccounts()
+
+  const deployArgs = {
+    from: deployer,
+    args: [],
+    log: true,
+  }
+  const ethOwnedResolver = await deploy('OwnedResolver', deployArgs)
+  if (!ethOwnedResolver.newlyDeployed) return
+}
+
+func.id = 'resolver'
+func.tags = ['resolvers', 'EthOwnedResolver']
```

Comment:
I've been using the convention of making one of the tags the contract name - so just `OwnedResolver`.

Reply:
Unless there's a compelling need for `EthOwnedResolver`, I would replace it.